.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
+  "eslint.validate": [
+    "javascript",
+    "javascriptreact",
```

Comment:
what is `javascriptreact`?

Reply:
It's copy pasted from here https://eslint.vuejs.org/user-guide/#editor-integrations

Reply:
Long answer: I don't know, I'm just going off their recommendation.